src/LinearSolvers/ColumnwiseLUSolver_kernels.jl

```diff
@@ -0,0 +1,411 @@
+using Requires
+@init @require CUDAnative = "be33ccc6-a3ff-5ff2-a52e-74243cff1e17" begin
+  using .CUDAnative
+end
+using StaticArrays
+
+"""
+    band_lu_knl!(A, Val(Nq), Val(Nqj), Val(nstate), Val(nvertelem),
```

Comment:
Signature doesn't match method below (`Val(Nq)` has been split it looks like)

Reply:
Thanks, fixed.